services/dask-sidecar/tests/unit/test_tasks.py

```diff
@@ -583,12 +583,11 @@ async def test_run_computational_sidecar_dask(
         TaskProgressEvent.parse_raw(msg).progress for msg in progress_sub.buffer
     ]
     # check ordering
-    assert worker_progresses == list(
+    assert worker_progresses == sorted(
```

Comment:
Just out of curiosity, why this change?

Reply:
so I found out in my test that
```python
set([0, 0.05, 0.95, 1.0]) == {0, 0.95, 0.05, 1.0}
```

Reply:
contrary to c++, python sets are unordered